### ISSUE TITLE
New wikis don't show the logo

### ISSUE THREAD

**Issue by MatmaRex** [MatmaRex/patchdemo#386]
e.g. https://patchdemo.wmflabs.org/wikis/a65f62738c/w/index.php/Main_Page

@edg2s Probably a bug from your config stuff

**Comment by MatmaRex** [comment]
Fixed by #387

**Comment by edg2s** [comment]
Looks like #387 (so no configs getting copied at all):
```
/var/www/html/wikis/a65f62738c/w/settings.d$ ls
feature-instantCommons.php
```
